contents/handbook/words-and-pictures/events.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Events
+sidebar: Handbook
+showTitle: true
+---
+
+> If you’re planning an event and want support or promotion from the PostHog team, go to [Community Events](https://posthog.com/handbook/words-and-pictures/events#community-events)
```

Comment:
```suggestion
> If you’re planning an event and want support or promotion from the PostHog team, go to [Community Events](/handbook/words-and-pictures/events#community-events)
```

Won't correct them all, but tiny FYI that internal links like this are best as relative paths